src/testSimulator/Calculator.java

```diff
@@ -0,0 +1,48 @@
+package testSimulator;
+
+import java.util.ArrayList;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Calculator {
+
+	private ArrayList<Double> Coefficients;
+	private double delta;
+
+	public ArrayList<Double> parseEquasionCoefficients(Classroom classroom) {
```

Comment:
Change the whole method. I don't want to see removing some random elements from the array anymore.